.gitignore

```diff
@@ -1,3 +1,32 @@
+# Node artifact files
+node_modules/
+dist/
+
+# Compiled Java class files
```

Comment:
Remove this, this is not a Java application

Reply:
Done